IbisLib/abstractpolydataobject.cpp

```diff
@@ -0,0 +1,515 @@
+/*=========================================================================
+Ibis Neuronav
+Copyright (c) Simon Drouin, Anna Kochanowska, Louis Collins.
+All rights reserved.
+See Copyright.txt or http://ibisneuronav.org/Copyright.html for details.
+
+     This software is distributed WITHOUT ANY WARRANTY; without even
+     the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+     PURPOSE.  See the above copyright notice for more information.
+=========================================================================*/
+#include <QFileDialog>
+#include <QMessageBox>
+#include "polydataobject.h"
+#include <vtkPolyData.h>
+#include <vtkPolyDataMapper.h>
+#include <vtkRenderer.h>
+#include <vtkActor.h>
+#include <vtkTransform.h>
+#include <vtkAssembly.h>
```

Comment:
vtkRenderer and vtkAssenly are not used in AbstractPolyDataObject

Reply:
vtkRenderer seems to be used
however vtkAssenly can be removed